Styling and positioning for landing page

Foadosman opened the pull request:
### What does this PR do?
completes styles on landing page.

### Description of Task to be completed?

### How should this be manually tested?

### Any background context you want to provide?

I had a bit of hard time finding the right positions. It's not easy finding the exact x/y position in the design.
Please give me feedback if this isn't good enough or if something should be changed.

Foadosman committed:
changes bg-color

Foadosman committed:
changes bg-color

Foadosman committed:
added styles and added positioning.

Foadosman committed:
added styles and added positioning.

Janicke92 committed:
Merge branch 'develop' into landing-page-styles-position

Janicke92 reviewed (dismissed):
Looks good! :)

kristinatonnessen commented on templates/index.html:
```diff
@@ -7,28 +7,34 @@
+                <button class="box-border border-2 border-secondary w-[119px] rounded-md text-secondary text-[26px] font-bold flex items-center justify-center hover:bg-secondary-hover">Log in</button>
+            </div>
+            </div>
         </section>
```
mobile version needs to be fixed :D 
![Skjermbilde 2025-01-15 kl  22 57 02](https://github.com/user-attachments/assets/91c65744-ed8f-4ec6-ac5b-4fe1a25cd285)
- footer, change to the footer version img
- buttons
- text
- img

kristinatonnessen replied:
Janicke cant see with live server due to an error abi is working on. thats why she approved and i didnt!

Foadosman replied:
Fixed buttons for mobile view. Still have to fix positioning for H1 and H2 on mobile and change the footer

Foadosman committed:
Merge branch 'landing-page-styles-position' of https://github.com/NoroffFEU/edu-gate into landing-page-styles-position

Foadosman committed:
added responsive styling for buttons

Janicke92 committed:
Merge branch 'develop' into landing-page-styles-position

Foadosman committed:
fixed prositon for h1 and h2 on mobile and added new  header

Foadosman committed:
Merge branch 'landing-page-styles-position' of https://github.com/NoroffFEU/edu-gate into landing-page-styles-position

Foadosman committed:
fixed position for <h1> & <p>, added new header and footer

Foadosman commented:
Fixed position and size for h1 and p1.
Added new header and footer.

These should be the final changes, please tell me if you see anything wrong or missing.

kristinatonnessen commented on templates/index.html:
```diff
@@ -7,30 +7,98 @@
+            <p class="text-[14px] sm:text-[16px] font-medium absolute bottom-[120px] sm:top-[95px]">Welcome to Edugate, the platform that lets you access your exam results online</p>
+            </div>
+            <div class="flex flex-col sm:absolute sm:bottom-[60px] sm:left-1/2 sm:transform sm:-translate-x-1/2 items-center sm:flex-row gap-4 mt-4">
+                <button class="bg-primary h-[40px] sm:h-[59px] w-[100px] sm:w-[133px] rounded-md text-neutral-light-gray text-[18px] sm:text-[26px] font-bold flex items-center justify-center hover:bg-primary-hover">Sign up</button>
```
From around 900px i would have the buttons placed like the mobile view due to low color contrast for the secondary button. check photo
![Skjermbilde 2025-01-31 kl  12 15 43](https://github.com/user-attachments/assets/8dff169a-3b0c-4c92-915a-77bdb77bbcca)
![Skjermbilde 2025-01-31 kl  12 16 47](https://github.com/user-attachments/assets/51491c99-5122-4406-ad21-ded39381f904)

Foadosman replied:
I see what you mean. This one is a bit challenging beacuse tailwind has spesific breakpoints. Like for example sm, md, lg etc. The closes brakpoint is md (768px).
I'll try to find a solution.

kristinatonnessen replied:
go with the larger break point if needed to accommodate to the accessibility quality

Foadosman replied:
I'll try lg as breakpoint

Foadosman replied:
I've added lg as breakpoint. I can also try to make the buttons move i bit down until it hits md or sm instead if you think that's a better idea.

kristinatonnessen commented on templates/index.html:
```diff
@@ -7,30 +7,98 @@
-        <img src="../public/img/circle-footer.png" alt="">
-    </footer>
+   </main>
+   <footer class="flex justify-end sm:justify-center">
```
Add margin top for footer mobile, or margin bottom for buttons/main
![Skjermbilde 2025-01-31 kl  12 17 27](https://github.com/user-attachments/assets/b3d331e5-89de-44f4-adc4-830336a6cc99)

Foadosman replied:
adding this

kristinatonnessen commented on templates/index.html:
```diff
@@ -7,30 +7,98 @@
+            <img src="../public/img/heroimage-edugate.png" alt="hero">
+            <div class="inset-0 flex flex-col items-center justify-center text-center">
+            <h1 class="text-neutral-dark-gray text-[32px] font-bold font-heading absolute top-[10px] sm:top-[40px]">Edugate</h1>
+            <p class="text-[14px] sm:text-[16px] font-medium absolute bottom-[120px] sm:top-[95px]">Welcome to Edugate, the platform that lets you access your exam results online</p>
```
Add more margin/padding to <p> for mobile view like in the design
<img width="283" alt="Skjermbilde 2025-01-31 kl  12 15 01" src="https://github.com/user-attachments/assets/5ad2ba24-6763-4be9-8faf-7ec682ece279" />

Foadosman replied:
I'll add margin on right and left side of the text

kristinatonnessen reviewed (changes requested):
just some small changes and we're good to go 😃

Janicke92 committed:
Merge branch 'develop' into landing-page-styles-position

kristinatonnessen commented:
Created issue #323 to document the changes needed to get done to this file, however the css isn't working for Fuad, Janicke or me in this branch, so there may be other changes as well

Anne-Zwift reviewed (dismissed):
@Foadosman looks like the footer are misplaced. Not sure if it's only on my screen.
![image](https://github.com/user-attachments/assets/6e224693-2fe1-49e8-9734-41628da17056)

Foadosman commented:
@Anne-Zwift 

I don't have the same issue. Are you sure that you have the latest updates?

Foadosman committed:
added margins and changed break point on buttins to lg

kristinatonnessen committed:
Merge branch 'develop' into landing-page-styles-position

kristinatonnessen reviewed (approved):
good work 😄
